.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,45 @@
+name: Create Release Pull Request
+
+on:
+  workflow_dispatch:
+    inputs:
+      base-branch:
+        description: 'The base branch for git operations and the pull request.'
+        default: 'main'
+        required: true
+      release-type:
+        description: "minor"
+        required: true
+        default: 'patch'        
+      release-version:
+        description: '1.1'
+        required: false
+
```

Comment:
It seems like there are some differences between this file and the module template — would you mind re-copying [this file](https://github.com/MetaMask/metamask-module-template/blob/main/.github/workflows/create-release-pr.yml)?